mergebot_poller_test.py

```diff
@@ -0,0 +1,131 @@
+import unittest
+import mergebot_poller
+from mock import patch
+
+
+class GithubPollerTest(unittest.TestCase):
+
+    @patch('mergebot_poller.GithubPoller')
+    def testCreatePollerSuccess(self, mock_poller):
+        # Tests creating a poller with a valid SCM type.
+        mergebot_poller.create_poller({'scm_type': 'github'})
+        self.assertEqual(mock_poller.call_count, 1)
+
+    def testCreatePollerFailure(self):
+        # Tests creating a poller with an invalid SCM type.
+        self.assertRaises(AttributeError, mergebot_poller.create_poller,
+                          {'scm_type': 'unicorn'})
+
+    @patch('github_helper.GithubPR')
+    def testCheckValidPR(self, mock_pr):
```

Comment:
These look pretty good. I'd consider splitting up multiple tests so that each test reads something like:

1. Setup mocks and return values
2. Interact with the code under test
3. Assert that the code produced the correct results
4. Assert that the code called the right mock methods/had the right side effects